orchestrator/consumers/chatbot/service.py

```
@@ -105,9 +105,10 @@ class ToolExecutionTracker:
     SEARCH_TOOLS = {
         'search_knowledge', 'semantic_search', 'search_codebase',
         'search_tables', 'search_images', 'search_formulas',
-        'search_multimodal'
-        # PRD-156 S3: smart_query_database / query_database removed — NL2SQL is
-        # off the chat surface until PRD-160 re-enables it workspace-scoped.
+        'search_multimodal',
+        # PRD-160 S1: NL2SQL re-enabled workspace-scoped & in-process. Treated as
+        # a search tool so semantically-similar repeat questions are deduped.
+        'smart_query_database', 'query_database',
     }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)